app_src/ui/screens/stats_screen.py

```
@@ -331,6 +331,20 @@ def refresh_storage_data(self,widget=None):
         cm2_file_path_size = os.path.getsize(cm2_file_path) if os.path.exists(cm2_file_path) else 0
         config_total_bytes=cm1_file_path_size+cm2_file_path_size
 
+        try:
+            from android_notify.internal.java_classes import autoclass
+            PythonActivity = autoclass("org.kivy.android.PythonActivity")
+            context = PythonActivity.mActivity.getApplicationContext()
+            prefs = context.getSharedPreferences("update_checker_prefs", 0)
+            prefs_file_path = context.getFilesDir().getParent() + "/shared_prefs/update_checker_prefs.xml"
+            prefs_file_size = os.path.getsize(prefs_file_path) if os.path.exists(prefs_file_path) else 0
+            config_total_bytes += prefs_file_size
+        except:
```

Ruff (0.16.2)

[error] Do not use bare `except`

(E722)